src/components/task-plan/reading.cjsx

```diff
@@ -150,45 +113,67 @@ ReadingPlan = React.createClass
     if plan?.due_at
       dueAt = new Date(plan.due_at)
 
-    footer= <PlanFooter id={id} courseId={courseId} />
-
-    <BS.Panel bsStyle="primary" className="create-reading" footer={footer} header={headerText}>
-      <div className="-reading-title">
-        <label htmlFor="reading-title">Title</label>
-        <input
-          ref="title"
-          id="reading-title"
-          type="text"
-          value={plan.title}
-          placeholder="Enter Title"
-          onChange={@setTitle} />
-      </div>
-      <div className="-reading-open-date">
-        <label htmlFor="reading-open-date">Open Date</label>
-        <DateTimePicker
-          id="reading-open-date"
-          format="MMM dd, yyyy"
-          time={false}
-          calendar={true}
-          readOnly={false}
-          onChange={@setOpensAt}
-          max={dueAt}
-          value={opensAt}/>
-      </div>
-      <div className="-reading-due-date">
-        <label htmlFor="reading-due-date">Due Date</label>
-        <DateTimePicker
-          id="reading-due-date"
-          format="MMM dd, yyyy"
-          time={false}
-          calendar={true}
-          readOnly={false}
-          onChange={@setDueAt}
-          min={opensAt}
-          value={dueAt}/>
-      </div>
-      <SelectTopics courseId={courseId} planId={id} selected={topics}/>
-    </BS.Panel>
+    footer = <PlanFooter id={id} courseId={courseId} />
+
+    if (@state?.showSectionTopics)
+      formClasses.push('hide')
+      selectReadings = <ChooseReadings
+                        hide={@hideSectionTopics}
+                        courseId={courseId} 
+                        planId={id} 
+                        selected={topics}/>
+
+    <div>
```

Comment:
I don't think it really needs one, as the panel below has a class already.  The div is just there to contain both the dialog and the panel.